browser/ui/speedreader/speedreader_tab_helper.cc

```diff
@@ -165,6 +165,14 @@ void SpeedreaderTabHelper::ProcessIconClick() {
   }
 }
 
+DistillState SpeedreaderTabHelper::PageDistillState() const {
+  auto* speedreader_service = GetSpeedreaderService();
+  if (!speedreader_service || !speedreader_service->IsFeatureEnabled()) {
```

Comment:
one more question - this service could be null in this tab helper?

Reply:
No. I'll remove the nullptr check